node/node.go

```diff
@@ -391,7 +407,8 @@ func (n *Node) createChronologyHandler(rounder consensus.Rounder) (consensus.Chr
 	chr, err := chronology.NewChronology(
 		n.genesisTime,
 		rounder,
-		n.syncTimer)
+		n.syncTimer,
+		n.appStatusHandler)
```

Comment:
Same here, apply what was commented on the Blockchain struct

Reply:
done